lib/src/screen/search_screen.dart

```diff
@@ -0,0 +1,59 @@
+import '../widgets/mock_genres.dart';
+import '../widgets/mock_movies.dart';
+import '../styles/dimensions.dart';
+import '../styles/styles.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+
+class SearchScreen extends StatefulWidget {
+  SearchScreen({
+    Key key,
+    this.title,
+    this.icon,
+  }) : super(key: key);
+
+  final String title;
+  final IconData icon;
+
+  @override
+  State<StatefulWidget> createState() => _SearchScreenState();
+}
+
+class _SearchScreenState extends State<SearchScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Icon(
+              widget.icon,
+            ),
+            Text(
+              widget.title,
+              style: new TextStyle(
+                fontSize: Dimension.searchScreenTextFS,
+                fontWeight: FontWeight.bold,
+                foreground: new Paint()..shader = linearGradient,
```

Comment:
it's not necessary to use the "new" keyword, you can remove it, beside that, LGTM

Reply:
Done, i removed the unnecessary 'new' keyword.